### DISCUSSION TITLE
flutterflow build for android and ios

### DISCUSSION THREAD

**Discussion by smoothmoves101** [codemagic-ci-cd/codemagic-docs#2420] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I am trying to build my android apk and ios ipa from my flutterflow project can you please enable ipa download from project build on codemagic i need to test my app

### Expected results

download my ipa and android apk from my project build from flutterflow 

### Actual results

only receiving the android apk file and all artifacts from the build and not the ios ipa file from it

### Build id (optional)

65f8c3bad5ceed65837fa35c, 65f9c906657a835b7497805e, 65f9c27b8a7a4fd2f4067606

**Comment by himesh-cm** [comment, marked as answer]
Hi @smoothmoves101, You should see an option to download the artifacts from your build details page below. Let me know if you still need any assistance.

<img width="404" alt="Screenshot 2024-03-25 at 17 56 15" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/106069365/38e18bdd-78f2-469b-9916-5324ec014c7d">

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
